src/components/AvatarSpace.vue

```diff
@@ -1,25 +1,30 @@
 <script setup lang="ts">
-import { withDefaults } from 'vue';
+import { sha256 } from 'js-sha256';
+import { withDefaults, computed } from 'vue';
 
-withDefaults(
+const props = withDefaults(
   defineProps<{
-    space: { id: string };
-    size: number;
+    space: { id: string; avatar: string };
+    size?: string;
   }>(),
   {
-    size: 22
+    size: '22'
   }
 );
+
+const avatarHash = computed(() => {
+  if (!props.space.avatar) return '';
+  const hash = sha256(props.space.avatar).slice(0, 16);
+  const hashWithoutLetters = hash.replace(/[^0-9]/g, '');
```

Comment:
Still not sure why this has to be numbers

Reply:
Yeah now it doesn't seem to make a difference anymore. I removed it